openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -28,9 +28,10 @@ <h2 class="step__title">
                     <article class="submission__answer__display step__content__section">
                         <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
 
-                        {% include "openassessmentblock/oa_submission_answer.html" with answer=student_submission.answer answer_text_label="Your response to the question above:" %}
+                        {% trans "Your response to the question above:" as translated_label %}
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=student_submission.answer answer_text_label=translated_label %}
 
-                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url header="Your Upload" class_prefix="submission__answer" %}
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url header="yours" class_prefix="submission__answer" %}
```

Comment:
I think this is pretty confusing. If we do decide to go with this approach, comments will be necessary to explain that "yours" is a flag, not a string displayed to the user.